charts/zabbix/values.yaml

```diff
@@ -440,6 +440,8 @@ zabbixProxy:
   # -- The kubelet uses startup probes to know when a container application has started.  Reference: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
   startupProbe: {}
   # -- nodeSelector configurations. Reference: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/
+  lifecycle: {}
```

Comment:
Hello @antonin-rouault!

Thanks for your contribution.
This line must be after the comment.

Can you change order of lines 443 and 444?